config/robot.urdf

```diff
@@ -195,25 +208,24 @@
               <stddev>0.007</stddev>
             </noise>
           </camera>
-          <plugin name="camera_controller" filename="libgazebo_ros_camera.so">
-            <ros>
+          <!-- <plugin name="camera_controller" filename="libgazebo_ros_camera.so"> -->
```

Comment:
I think we can remove this and bridge the relevant camera topics from Gazebo. Maybe @WilliamLewww can provide pointers for anything we should add to this file and/or topics that can be bridged.